### PR TITLE
Review project

### PR DESCRIPTION
**Pull request by maaloum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,5 @@
...
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (maaloum) [inline README.md]: Kindly, customise your README file in order to give more details about the project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+  </body>
+</html>
```
Comment (maaloum) [inline index.html:13]: Please do not leave unused code in your code. It will make your code less clean.

**Review by maaloum — COMMENTED** [review]
Please add linters to your project.

**Comment by maaloum** [comment]
In this project I add :
- Index.html
- style.css
- index.js
- Configure linters.

**Review by maaloum — COMMENTED** [review]
The pull request does not have a description which can let it invalid. Kindly, add a description to your PR

**Comment by codecaiine** [comment]
Hi @maaloum , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good use of inline comments :+1: 
- Good use of workflow :heavy_check_mark: 
- Comments are polite :+1: 

## Required Changes ♻️

- [x] It is to have mentioned linters, as soon as there are errors or configurations are missing for this particular project, please mark the project as invalid because liners is part of the requirement. 
- [x] You also mentioned that there was no pr description and that it might be invalid. but actually when the description is missing, please mark the project invalid.

_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL]  I would like to recommend you use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by maaloum** [comment]
While the great effort, you put into this project. But it is invalid PR because you should add Linters and a description to it.

**Review by zhadier — APPROVED** [review]
## Hi @maaloum  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**